scipy/sparse/_base.py

```diff
@@ -1146,6 +1158,17 @@ def sum(self, axis=None, dtype=None, out=None):
         # We use multiplication by a matrix of ones to achieve this.
         # For some sparse array formats more efficient methods are
         # possible -- these should override this function.
+
+        # if 2-d and axis is tuple
+        if self.ndim == 2 and type(axis) is tuple and len(axis) == 2:
+            # adjust axes
+            axis = [ax if ax>=0 else ax+2 for ax in axis]
```

Comment:
I think this does what you want without requiring changes later for nD case
```suggestion
        # if axis is tuple of all dimensions then same as axis=None
        if type(axis) is tuple and len(axis) == self.ndim:
            # adjust axes
            axis = [ax if ax >= 0 else ax + self.ndim for ax in axis]
```